### ISSUE TITLE
Marking of series / Маркировка серий

### ISSUE THREAD

**Issue by yuriy-zhilovets2** [PastVu/pastvu#515]
Многие фотографии идут сериями - это могут быть, например, наборы открыток или фотографии, сделанные автором одновременно. Желательно такие серии как-то объединять вместе, чтобы по одному фото можно было найти все остальные.
Сейчас это делается или указанием одинакового описания или в комментариях, что выглядит неуклюже.

Чтобы не плодить лишних сущностей, проще всего привязывать фото к первому фото серии и затем в отдельном поле автоматически показывать ссылки на остальные фото серии. Не повредит также описание серии, поскольку оно часто не совпадает с описанием конкретной фотографии.

**Comment by kabalin** [comment]
Это через систему тэгов планируется реализовать https://github.com/PastVu/pastvu/discussions/20

**Comment by yuriy-zhilovets2** [comment]
> Это через систему тэгов планируется реализовать #20

Многие дают дополнительную информацию, например, для набора открыток это выходные данные. Поместится ли это в тегах?

**Comment by kabalin** [comment]
> Многие дают дополнительную информацию, например, для набора открыток это выходные данные. Поместится ли это в тегах?

Так как система тэгов находится на этапе планирования, дополнительный функционал можно обсудить и добавить при реализации. Скажем тэг может иметь поле описания где можно указать дополнительные данные. К примеру тэг может быть "Набор открыток Воронеж 1963", а в описании уже подробная информация с выходными данными и издательством. В любом случае такая классификация имеет много общего с тэгами - если даже "серию" связывать каким-то иным образом в интерфейсе, целесообразно базировать ее на системе тэгов.

**Comment by paul-k-pastvu** [comment]
Да, технологически это всё должен быть один механизм. А дальше теги будут, вероятно, типизованы, и для обработки отдельных типов тегов будут какие-то свои интерфейсные решения. Скажем, нынешние поля "Автор", "Источник" (и даже "Адрес") скорее всего тоже "уйдут" в систему тегов.
